docker/apps/README.md

```diff
@@ -0,0 +1,23 @@
+# `docker-apps`
+
+This directory contains a generic dockerfile that accepts two arguments: `TAG` and `BIN_TARGET`. It clones the `kona`
+repository at the specified `TAG`, and builds the `BIN_TARGET` with the `release` profile, before finally copying it
+into an `ubuntu:22.04` image with the binary in the PATH and set as the image's entrypoint.
+
+Adding a new binary to the `docker-bake.hcl` in the parent directory is as follows:
+
+```hcl
+target "<bin-name>" {
+  inherits = ["docker-metadata-action"]
+  context = "."
+  dockerfile = "docker/apps/kona_app_generic.dockerfile"
+  args = {
+    TAG = "${GIT_REF_NAME}"
+    BIN_TARGET = "<bin-name>"
+  }
+  platforms = split(",", PLATFORMS)
+}
+```
```

Comment:
Do we actually need to do this? Or can we make the target "kona-generic-apps" and then use an input arg in the tag as the `BIN_TARGET`? Then we don't need to change any code at all for new apps, we can just pass the bin name in as a variable in the tag.

Reply:
Will be a future enhancement